consul/aio.py

```diff
@@ -42,8 +43,10 @@ def get(self, callback, path, params=None):
         uri = self.uri(path, params)
         return self._request(callback, 'GET', uri)
 
-    def put(self, callback, path, params=None, data=''):
+    def put(self, callback, path, params=None, data='', json=None):
```

Comment:
Note that this is a wrapper method to requests and the `json` parameter is not passed through to requests, it is dumped to string and passed through as data. So I do not think the json parameter adds anything here or in the other implementations at the moment.